code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine_vr.dm

```diff
@@ -62,15 +62,20 @@
 /datum/reagent/vermicetol
 	name = "Vermicetol"
 	id = "vermicetol"
-	description = "A potent chemical that treats burn damage at an exceptional rate and lasts a while."
+	description = "A potent chemical that treats all damage at an exceptional rate and lasts a while."
 	taste_description = "sparkles"
 	reagent_state = SOLID
 	color = "#964e06"
-	overdose = 10
+	overdose = 15
 	scannable = 1
-	metabolism = 0.02
+	metabolism = 0.05
 	mrate_static = TRUE
 
 /datum/reagent/vermicetol/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
 	if(alien != IS_DIONA)
-		M.heal_organ_damage(0, 110 * removed) //Not as potent as Kelotane, but lasts LONG.
+		var/chem_effective = 1
+		if(alien == IS_SLIME)
+			chem_effective = 0.5
+		M.heal_organ_damage(60 * removed, 60 * removed * chem_effective)
+		M.adjustOxyLoss(-120 * removed * chem_effective)
+		M.adjustToxLoss(-60 * removed * chem_effective)
```

Comment:
Lemme talk to admins about this part. This part has some 'rezadone-like' implications.

Reply:
It heals 3 brute/burn/toxin per tick, and 6 oxy per tick, due to the very, very slow metabolization, which is why it is more super tricordazine than rezadone which heals a huge amount in a very short time.

Reply:
Actually in talking to Temp and doing the math, a single piece of grubmeat is enough to give you 12 minutes of massive healing. This should be nerfed, or the recipe should be made more expensive (or both)

Reply:
Recommending nerf, speed up metab rate and decrease healing so it'll give maybe like... 2 per tick max and not nearly for as long. It might actually make vermicetol useful enough to make.

Alternatively, making it the 8 heal brute chem could be considered as well. Solar grubs happen what feels like nearly every round. The only issue is that I'm not sure if you can butcher the solar grub larvae and they tend to get killed pretty quickly - before they mature. But that's an IC thing that can probably be worked out.